### PR TITLE
RD-1833 Filters docs

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

**Review by tehasdf — APPROVED** [review]

File: content/cli/orch_cli/blueprints.md [hunk 1]
```diff
@@ -456,3 +479,172 @@ Blueprint `cloudify-nodecellar-example` was set to global
...
+* `--attrs-filter TEXT`     A list of attributes' filter rules separated
+                            with an `and`. Attributes' filter rules must
+                            be one of:  `<key>=<value>`, `<key>!=<value>`,
+                            `<key> contains <value>`, `<key> does-not-
+                            contain <value>`, `<key> starts-with <value>`,
+                            `<key> ends-with <value>`. `<key> is not empty`.
+                            `<value>` can be a single string, or a list of
+                            strings of the form `[<value1>,<value2>,...]`.
+                            Allowed attributes to filter by are:
+                            [blueprint_id, created_by, site_name, schedules].
```
Comment (glukhman) [inline content/cli/orch_cli/blueprints.md:520]: Looks like these are the attributes for deployment filters, rather than blueprints'

Comment (YoniItzhak) [reply content/cli/orch_cli/blueprints.md]: Right, changed.

**Review by glukhman — APPROVED** [review]

File: content/cli/orch_cli/deployments.md [hunk 2]
```diff
@@ -850,3 +874,174 @@ Node instances before rollback:
...
+* `-s, --pagination-size INTEGER`   The max number of results to retrieve per page [default: 1000]
+
+
+#### filters update
+
+##### Usage
+
+`cfy deployments filters update [OPTIONS] FILTER_ID` 
+
+Update an existing deployments' filter's filter rules or visibility.
```
Comment (ofercloudify) [inline content/cli/orch_cli/deployments.md:1016]: Same comment as above - worth mentioning how the update takes place (what is updated)

**Review by ofercloudify — CHANGES_REQUESTED** [review]

File: content/cli/orch_cli/blueprints.md [hunk 3]
```diff
@@ -456,3 +479,171 @@ Blueprint `cloudify-nodecellar-example` was set to global
...
+* `-s, --pagination-size INTEGER`   The max number of results to retrieve per page [default: 1000]
+
+
+#### filters update
+
+##### Usage
+
+`cfy blueprints filters update [OPTIONS] FILTER_ID` 
+
+Update an existing blueprints' filter's filter rules or visibility.
```
[the remark this answers is not in the document]
Comment (YoniItzhak) [reply content/cli/orch_cli/blueprints.md:619]: Changed to:
"Any flag provided as part of the update (labels' filter-rules / attrbiutes' filter-rules / visiblility) overrides only the corresponding value.   
E.g. if only the flag `--labels-rules` is provided, the labels' filter-rules will be overriden, but the visibility and attributes' filter-rules of the filter will stay the same."

File: content/cli/orch_cli/blueprints.md [hunk 4]
```diff
@@ -456,3 +479,171 @@ Blueprint `cloudify-nodecellar-example` was set to global
 
 ...
 {{< /highlight >}}
+
+
+### filters
+
+A filter is defined as a set of filter-rules that can be used to filter a list of blueprints, based on their labels and certain attributes.
+Blueprints can be filtered by the attribute `created_by`.
```
[the remark this answers is not in the document]
Comment (YoniItzhak) [reply content/cli/orch_cli/blueprints.md:487]: Changed to:
"At the moment, the supported blueprint attributes to filter by include only `created_by`."

File: content/cli/orch_cli/deployments.md [hunk 5]
```diff
@@ -235,6 +235,30 @@ If `--blueprint-id` is provided, list deployments for that blueprint.
 
 *  `-b, --blueprint-id TEXT` -
                         The ID of the blueprint for which you want to list deployments.
+   
+* `--filter-id TEXT`    Filter results according to the specified
+                        filter
```
[the remark this answers is not in the document]
Comment (YoniItzhak) [reply content/cli/orch_cli/deployments.md:240]: Changed also for blueprints.

File: content/cli/orch_cli/deployments.md [hunk 6]
```diff
@@ -850,3 +874,174 @@ Node instances before rollback:
 Added node instances:
         - vm1_b0kgj5 (vm1)
 {{< /highlight >}}
+
+
+### filters
```
[the remark this answers is not in the document]
Comment (YoniItzhak) [reply content/cli/orch_cli/deployments.md:879]: Added "Deployment" to all filters commands.

File: content/cli/orch_cli/blueprints.md [hunk 7]
```diff
@@ -456,3 +479,171 @@ Blueprint `cloudify-nodecellar-example` was set to global
 
 ...
 {{< /highlight >}}
+
+
+### filters
```
[the remark this answers is not in the document]
Comment (YoniItzhak) [reply content/cli/orch_cli/blueprints.md:484]: Added "Blueprint" to all filters commands.

File: content/cli/orch_cli/deployments.md (hunk 2 above)
Comment (YoniItzhak) [reply content/cli/orch_cli/deployments.md:1016]: Done.

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**